### DISCUSSION TITLE
Show some example of: Lambda, Tables, Images, TailWind

### DISCUSSION THREAD

**Discussion by greeto** [spatie/laravel-pdf#51] [Q&A]
Can you show example of:
Long Tables with Headers, Images, Images inside Table Cells, Full usage with Tailwind, How to put on Lambda?

**Comment by freekmurze** [comment]
I do have an example of using Tailwind: https://freek.dev/2639-creating-beautiful-invoices-using-tailwind-and-laravel-pdf

As for the rest, you'll have to research that yourself
